models/userDao.js

```diff
@@ -24,17 +26,24 @@ const signUp = async (user) => {
   }
 }
 
-const getEmail = async (email) => {
+const getUser = async (email) => {
   const rawQuery = `
-    SELECT email FROM users WHERE email = ?;`
+    SELECT
+      id,
+      first_name,
+      last_name,
+      email,
+      password,
+      phone_number,
+      birthday,
+      point
+      FROM users WHERE email = ?;`
```

Comment:
마이너한 부분이지만 들여쓰기 수정을 통해 쿼리문에서도 일관된 포매팅을 해주면 좋을 것 같습니다~!

```suggestion
    SELECT
      id,
      first_name,
      last_name,
      email,
      password,
      phone_number,
      birthday,
      point
    FROM users WHERE email = ?;`
```

Reply:
반영했습니다